src/ontology/reports/README.md

```diff
@@ -0,0 +1,54 @@
+## Codebook: `src/ontology/reports/`
+### 1. `mirror_signature-<ONTOLOGY_NAME>.tsv`
+A "mirror signature" file, which contains a list of class URIs from the unaltered source ontology.
+- `?term` (`uri`): URI for class in ontology.   
+
+Created by running `cd src/ontology; sh run.sh make reports/mirror_signature-<ONTOLOGY_NAME>.tsv`.
+### 2. `component_signature-<ONTOLOGY_NAME>.tsv`
+A "component signature" file, which contains a list of class URIs from Mondo's "alignment module" for the ontology. An alignment module is the list of classes we care about (e.g. all diseases).
+- `?term` (`uri`): URI for class in ontology.
+
+Created by running `cd src/ontology; sh run.sh make reports/component_signature-<ONTOLOGY_NAME>.tsv`.
+
+### 3. `<ONTOLOGY_NAME>_term_exclusions.txt`
+A simplified variation of (3). The final list of terms to exclude from Mondo for the given ontology. A simple, line-break-delimited list of terms, e.g. a single column with no column header. 
```

Comment:
(3)? Wrong numbering? Should be (4) right?

Reply:
Ah yes. I moved these over to another file and renumbered them. I forgot that these numbers were referenced! Convenient but error-prone. I'll fix.